pkgs/applications/misc/swappy/default.nix

```
@@ -0,0 +1,45 @@
+{ stdenv, fetchFromGitHub
+, meson
+, ninja
+, wayland
+, cairo
+, pango
+, gtk
+, pkgconfig
+, cmake
+, scdoc
+, libnotify
+, gio-sharp
+, glib
+}:
+
+stdenv.mkDerivation rec {
+  name = "swappy-${version}";
+  version = "1.2.0";
+
+  src = fetchFromGitHub {
+    owner = "jtheoof";
+    repo = "swappy";
+    rev = "v${version}";
+    sha256 = "13ky6nx6lprsiza62syqrq63i78fcg74lzd9pwa4kabjnqc350j5";
+  };
+
+  nativeBuildInputs = [ glib meson ninja pkgconfig cmake scdoc ];
+
+  buildInputs = [ cairo pango gtk libnotify wayland glib ];
+
+  strictDeps = true;
+
+  mesonFlags = [
+    # TODO: https://github.com/NixOS/nixpkgs/issues/36468
+    "-Dc_args=-I${glib.dev}/include/gio-unix-2.0"
+  ];
+
+  meta = {
+    homepage = "https://github.com/jtheoof/swappy";
+    description = "A Wayland native snapshot editing tool, inspired by Snappy on macOS ";
+    license = stdenv.lib.licenses.mit;
+    maintainers = [ stdenv.lib.maintainers.matthiasbeyer ];
+    platforms = with stdenv.lib.platforms; linux;
```

personal preference:
```suggestion
  meta = stdenv.lib; {
    homepage = "https://github.com/jtheoof/swappy";
    description = "A Wayland native snapshot editing tool, inspired by Snappy on macOS ";
    license = licenses.mit;
    maintainers = [ maintainers.matthiasbeyer ];
    platforms = with platforms; linux;
```